### PR TITLE
Prettier husky

### PR DESCRIPTION
**Pull request by Bashamega**
Hello:)

I have integrated prettier in husky

### CODE REVIEW

**Comment by Bashamega** [comment]
Hello @eddiejaoude :) Can you please merge this pr

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you. Please exclude `CHANGELOG.md` file as this is autogenerated and will cause conflicts

**Comment by eddiejaoude** [comment]
> Hello @eddiejaoude :) Can you please merge this pr

the extra notifications is not required

@Cahllagerfeld are you able to review?

**Comment by Bashamega** [comment]
> the extra notifications is not required

Sorry for the notification

**Comment by eddiejaoude** [comment]
thank you for understanding 👍

**Comment by Bashamega** [comment]
> Thank you. Please exclude `CHANGELOG.md` file as this is autogenerated and will cause conflicts

It should be excluded because of the prettier ignore.

**Comment by eddiejaoude** [comment]
> > Thank you. Please exclude `CHANGELOG.md` file as this is autogenerated and will cause conflicts
> 
> It should be excluded because of the prettier ignore.

Oh strange, there are changes to the file https://github.com/EddieHubCommunity/good-first-issue-finder/pull/415/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed

**Comment by eddiejaoude** [comment]
Strange why the ignore file is not being used, here are the official docs and the filename is correct https://prettier.io/docs/en/ignore.html

**Comment by Bashamega** [comment]
> Strange why the ignore file is not being used, here are the official docs and the filename is correct https://prettier.io/docs/en/ignore.html

Is it because we are using husky? I don't know how to fix it

**Comment by eddiejaoude** [comment]
> Is it because we are using husky? I don't know how to fix it

I don't think they are related or causing any issues. I see you have joined EddieHub Discord, let's continue the chat there and publish the conclusion here afterwards

**Comment by Bashamega** [comment]
Hello:)

I have fixed the issue. The issue was that instead of specifying .prettierignore we were specifying the .gitignore. I have now fixed it.

File: .husky/pre-commit [hunk 1]
```diff
@@ -3,4 +3,4 @@
 
 npx lint-staged
 
-npx prettier --ignore-path .gitignore --check --plugin prettier-plugin-svelte .
\ No newline at end of file
+npx prettier --ignore-path .prettierignore --check --plugin prettier-plugin-svelte .
```
Comment (eddiejaoude) [inline .husky/pre-commit]:
```suggestion
npx prettier --ignore-path .prettierignore --check --plugin prettier-plugin-svelte .

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -35,8 +34,9 @@
 
 ### Bug Fixes
 
-
 * caprover config ([38521e0](https://github.com/EddieHubCommunity/good-first-issue-finder/commit/38521e035edc251675ad28f5914aac477c4c5fae))
 
 
 
+
+
```
Comment (eddiejaoude) [inline CHANGELOG.md]: Please ignore this file as it is auto generated

**Pull request closed without merging**